README.md

```diff
@@ -14,148 +14,80 @@ Metro Board Reports is a member of the [Councilmatic family](https://www.council
 
 ## Setup
 
-**Install OS level dependencies:**
+These days, we run apps in containers for local development. Prefer to run the app locally? See the [legacy setup instructions](https://github.com/datamade/la-metro-councilmatic/blob/b8bc14f6d90f1b05e24b5076b1bfcd5e0d37527a/README.md).
```

Comment:
For additional context, a link to the work you have done about our new practices for dockerizing development would be super.